### PR TITLE
Add new line to end of file when rendering

### PR DESCRIPTION
**Pull request by ben-hudson**
Closes https://github.com/rhgrant10/tsplib95/issues/21

### CODE REVIEW

**Comment by ben-hudson** [comment]
Hey @PatrickOHara, if you're interested in merging your PR could you fix the conflicts with my fork?

**Comment by PatrickOHara** [comment]
Hi @ben-hudson, its been a few years since I looked at this code and I'm not sure which branch is correct. Any ideas?

p.s. I also made a fix on this PR which you might like to merge in: https://github.com/rhgrant10/tsplib95/pull/19

File: tsplib95/transformers.py [hunk 1]
```diff
@@ -244,14 +244,6 @@ def pack(self, items):
...
         return list(container)
 
-class TupleT(ContainerT):
-    """Transformer for tuples"""
-
-    def pack(self, items):
-        return tuple(items)
-
-    def unpack(self, container):
-        return tuple(container)
```
Comment (PatrickOHara) [inline tsplib95/transformers.py:254]: I can't remember the reason why I included this class, but it has something to do with this PR: https://github.com/rhgrant10/tsplib95/pull/19

**Comment by ben-hudson** [comment]
> Hi @ben-hudson, its been a few years since I looked at this code and I'm not sure which branch is correct. Any ideas?

I have no idea either, I can take a look at the merge conflicts and take my best guess. Do you remember what this PR was for?

I have your other PR (the edge_list one) since I only forked recently

File: tsplib95/fields.py [hunk 2]
```diff
@@ -238,7 +238,7 @@ class AdjacencyListField(TransformerField):
     def build_transformer(cls):
         return T.MapT(key=T.FuncT(func=int),
                       value=T.ListT(value=T.FuncT(func=int)),
-                      sep="\n",
+                      sep=('-1', ' -1\n'),
```
Comment (ben-hudson) [inline tsplib95/fields.py:241]: I'm not sure what the implication is, but the conflict is on this line. I have
```
value=T.ListT(value=T.FuncT(func=int), terminal='-1'),
sep='\n',
```
and you have
```
value=T.ListT(value=T.FuncT(func=int)),
sep=('-1', ' -1\n'),
```

Subtle difference but I don't know what the implication is... I think I made the change so add a new line at the end of file when rendering

**Pull request closed without merging**